sets_dbs/dp_infos/copy/sqlite_to_sql_format.py

```
@@ -0,0 +1,35 @@
+with open('backup.sql', 'w') as f:
+    # Write table creation statement
+    f.write("CREATE TABLE infos (url TEXT, urlid TEXT, file TEXT);\n")
+
+    # Iterate over data in chunks and write INSERT statements
+    chunk_size = 100000
+    num_chunks = (total_rows // chunk_size) + 1
+    for i in tqdm(range(num_chunks), total=7083106):
+        start = i * chunk_size
+        end = (i + 1) * chunk_size
+        df = pd.read_sql_query(f"SELECT * FROM infos LIMIT {start}, {chunk_size}", conn_sqlite)
+        insert_statements = df.apply(lambda row: f"INSERT INTO infos (url, urlid, file) VALUES ('{row['url']}', '{row['urlid']}', '{row['file']}');\n", axis=1)
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)